packages/database-typeorm/src/typeorm.ts

```diff
@@ -166,11 +166,13 @@ export class AccountsTypeorm implements DatabaseInterface {
   }
 
   public async findUserByServiceId(serviceName: string, serviceId: string): Promise<User | null> {
-    const service = await this.serviceRepository.findOne({
-      name: serviceName,
-      serviceId,
-      cache: this.options.cache,
-    } as any);
+    const service = await this.serviceRepository.findOne(
```

Comment:
Should this follow the same pattern than line 111? (with the where condition)
```
    const service = await this.serviceRepository.findOne({
      where: {
        name: 'email.verificationTokens',
        token,
      },
      cache: this.options.cache,
    });
```

Reply:
Well, both patterns are used in this file. Personally, I prefer the where clause pattern (like you suggested). I will change it